forms/tests/test_api.py

```diff
@@ -145,3 +146,38 @@ def test_answer_post(admin_client, admin_user, basic_form):
     response = admin_client.post(url, data=payload)
 
     assert response.status_code == 201
+    assert len(Entry.objects.all()) == 4
+
+    url = reverse("answer-detail", kwargs={'pk': 1})
+    payload = {
```

Comment:
tip: you could use the payload from the post response and just edit one entry or something like that. No need to redefine the whole structure.